test/OpenTelemetry.Instrumentation.AspNetCore.Tests/BasicTests.cs

```diff
@@ -663,14 +660,9 @@ public async Task HttpRequestMethodIsSetAsPerSpec(string originalMethod, string
     {
         var exportedItems = new List<Activity>();
 
-        var configuration = new ConfigurationBuilder()
```

Comment:
Is this related to this PR or something accidentally creeped into this ?

Reply:
its leftover clean up from https://github.com/open-telemetry/opentelemetry-dotnet/pull/5066